packages/video_player/video_player/lib/video_player.dart

```diff
@@ -299,8 +299,12 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
 
   /// Attempts to open the given [dataSource] and load metadata about the video.
   Future<void> initialize() async {
-    _lifeCycleObserver = _VideoAppLifeCycleObserver(this);
-    _lifeCycleObserver.initialize();
+    final allowBackgroundPlayback =
```

Comment:
nit: The linter isn't flagging this for omitting the datatype?

Reply:
We haven't converted video_player off of the legacy options yet unfortunately.